1087/en.md

```diff
@@ -0,0 +1,126 @@
+# [http://lightoj.com/volume_showproblem.php?problem=1087](http://lightoj.com/volume_showproblem.php?problem=1087 "1087 - Diablo") #
```

Comment:
Please add the problem title in this format - https://github.com/lightoj-dev/problem-tutorials/pull/11/files#diff-49d89fcefb3733849902cae167206bfef3d2d2fa1569a2abcc5b303e53f828b9R1